src/main/java/net/refsonyak/endgameaddons/Config.java

```diff
@@ -0,0 +1,9 @@
+public class Config {
+    /*Class that contains all configurable variables for quick'n'easy balance changes
+      To use, simply create a class in here, give it some variables, import it into your
+      file. Call on it using Config.<yourclass>.<yourvariable> and voila!*/
+
+    public class dropRates {
```

Comment:
This is horrible naming conventions and classes should be `static`. You also do not need inner classes for everything. Static fields should be enough. Please also provide information about de/serialization and configuration screens.